header.php

```diff
@@ -62,14 +62,14 @@
 ?>
 
   <!-- start content -->
-  <header id="header" class="container">
-    <div class="row">
+  <header id="header" class="container margin-top-small">
+    <div class="row align-center">
 
-      <div class="col col2">
+      <div id="header-logo" class="col col-s col-s-4 offset-s-4 col-m col-m-2 offset-m-0 col-l">
       <?php
-        $logo_id = IGV_get_option('_igv_metadata_logo_id');
+        $logo_id = IGV_get_option('_igv_header_logo_id');
 
-        if ($logo_id) {
+        if (!empty($logo_id)) {
```

Comment:
I dont think we should totally avoid hardcoding design assets. just put some stuff in settings that saves us asking client for, or text that will want editing